packages/mask/src/plugins/Trader/SNSAdaptor/trader/ConfirmDialog.tsx

```diff
@@ -67,42 +114,161 @@ export function ConfirmDialogUI(props: ConfirmDialogUIProps) {
     }, [trade, executionPrice])
     //#endregion
 
+    //#region gas price
+    const nativeToken = createNativeToken(chainId)
+    const tokenPrice = useNativeTokenPrice(chainId)
+
+    const gasFee = useMemo(() => {
+        return gas && gasPrice ? new BigNumber(gasPrice).multipliedBy(gas).integerValue().toFixed() : 0
```

Comment:
To return the same value type.

```suggestion
        return gas && gasPrice ? new BigNumber(gasPrice).multipliedBy(gas).integerValue().toFixed() : '0'
```